core/controllers/admin.py

```diff
@@ -854,3 +854,23 @@ def post(self):
             self.render_json({})
         else:
             raise self.InvalidInputException('This app cannot send emails.')
+
+
+class InteractionsByExplorationId(base.BaseHandler):
```

Comment:
InteractionsByExplorationId**Handler**

Reply:
@DubeySandeep other than this all good ?